test/lib/actions/RunAction.test.ts

```diff
@@ -0,0 +1,56 @@
+import path = require('path');
+import {Logger} from '@salesforce/core';
+import {expect} from 'chai';
+
+import {Results} from '../../../src/lib/output/Results';
+import {Inputs, RuleResult} from '../../../src/types';
+import {RunAction} from '../../../src/lib/actions/RunAction';
+import {InputProcessorImpl} from '../../../src/lib/InputProcessor';
+import {RuleFilterFactoryImpl} from '../../../src/lib/RuleFilterFactory';
+import {RunEngineOptionsFactory} from '../../../src/lib/EngineOptionsFactory';
+
+import {initializeTestSetup} from '../../test-related-lib/TestOverrides';
+import {FakeDisplay} from '../FakeDisplay';
+import {FakeResultsProcessorFactory, RawResultsProcessor} from './fakes';
+
+
+describe('Misc scanner run tests', () => {
+	const PATH_TO_CODE_FIXTURES = path.join('.', 'test', 'code-fixtures');
+
+	let logger: Logger;
+	let display: FakeDisplay;
+	let inputProcessor: InputProcessorImpl;
+	let resultsProcessor: RawResultsProcessor;
+	let runAction: RunAction;
+
+	beforeEach(async () => {
+		initializeTestSetup();
+		logger = await Logger.child('RunAction.test.ts');
+		display = new FakeDisplay();
+		inputProcessor = new InputProcessorImpl('testing', display);
+		resultsProcessor = new RawResultsProcessor();
+		runAction = new RunAction(
+			logger,
+			display,
+			inputProcessor,
+			new RuleFilterFactoryImpl(),
+			new RunEngineOptionsFactory(inputProcessor),
+			new FakeResultsProcessorFactory(resultsProcessor)
+		);
+	});
+
+	it('We can successful call the retire engine and get a violation', async () => {
```

Comment:
I didn't catch the issue with the new retire 4.x not having a bin directory because our unit tests do not actually call through to retire js. So I'm adding in this test to make sure that we call the subprocess at least once with yarn test so we catch locally before creating PR.